unit-tests/live/frames/test-backend-vs-frame-timestamp.py

```diff
@@ -0,0 +1,98 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2025 Intel Corporation. All Rights Reserved.
+
+# test:device D400* !D457
+
+import pyrealsense2 as rs
+from rspy import test
+import time
+
+dev, _ = test.find_first_device_or_exit()
+depth_sensor = dev.first_depth_sensor()
+is_global_time_enabled_orig = False
+fps = 30
+has_at_least_one_frame_arrived = False
+
+
+def check_backend_ts_greater_than_frame_ts(frame):
+    global has_at_least_one_frame_arrived
+    has_at_least_one_frame_arrived = True
+    backend_ts_supported = frame.supports_frame_metadata(rs.frame_metadata_value.backend_timestamp)
+    test.check(backend_ts_supported)
+    frame_ts = frame.get_frame_timestamp()
+    backend_ts = frame.get_frame_metadata(rs.frame_metadata_value.backend_timestamp)
+    delta = backend_ts - frame_ts
+    if not test.check(delta > 0):
+        print("frame_ts = " + repr(frame_ts) + ", backend_ts = " + repr(backend_ts) + ", delta = " + repr(delta) + ", this should be positive")
```

Comment:
We will get here on an error?
I think LibCI has some trach method for that no?

Reply:
Why not using log.e instead of print?

Reply:
ok